src/core/server/pulse/collectors/deployment.ts

```diff
@@ -17,8 +17,16 @@
  * under the License.
  */
 
-export interface IPulseElasticsearchClient {
-  createIndexIfNotExist?: (channel: string, mappings: Record<string, any>) => Promise<void>;
-  index: (channel: string, doc: Record<string, any>) => Promise<void>;
-  search: <T>(channel: string, query: Record<string, any>) => Promise<T[]>;
+import { PulseCollector } from '../types';
```

Comment:
Oh! I don't know why git considered this a file moved. 
The real action was:
1. Create the `collectors/deployment.ts` for my channel. It shouldn't cause any issues on your end.
2. Remove `client_wrappers/types.ts` because we only needed the interface when we were creating the implementations for it in both: public and server. But now, we only have the `PulseElasticsearchClient` in the server. So every time we want to maintain the client, we'll need to also update the types in the interface, duplicating the amount of work to make this evolve.